src/benchcab/benchcab.py

```diff
@@ -202,6 +202,7 @@ def fluxsite_submit_job(self, config_path: str, skip: list[str]) -> None:
                 modules=config["modules"],
                 pbs_config=config["fluxsite"]["pbs"],
                 skip_bitwise_cmp="fluxsite-bitwise-cmp" in skip,
+                verbose=logger.getEffectiveLevel() == logging.DEBUG,
```

Comment:
I thought we had already created a utility function in utils.py that did this, but apparently not.

I can see us wanting to do this in multiple places down the track so I'd suggest writing a quick function in utils.

```python
def is_verbose():
    return get_logger().getEffectiveLevel() == logging.DEBUG
```

Then using that function here.